.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up dotnet ${{ matrix.dotnet-version }}
+        uses: actions/setup-dotnet@v3
+      - name: Set up msbuild
+        uses: microsoft/setup-msbuild@v2
+      - name: Set up nuget
+        uses: nuget/setup-nuget@v2
+        with:
+          nuget-version: '5.x'
+      - name: Run mnsbuild
+        run: msbuild trino-csharp/TrinoDriver.sln   
+      - name: Run nuget for client
+        run: nuget pack Trino.Client\Trino.Client.nuspec
```

Comment:
This should go into a release workflow.

Reply:
I thought we just set it here for starters .. but fine.. removed for now.. one step at a time.